File: 2016. Maximum Difference Between Increasing Elements.go
func maximumDifference(nums []int) int {
    max := 0
	len := len(nums)
	for i := 0; i < len-1; i++ {
		for j := i + 1; j < len; j++ {
			if max < nums[j]-nums[i] {max = nums[j] - nums[i]}
		}
	}
	if max == 0 {return -1}
	return max
}

File: 2016. Maximum Difference Between Increasing Elements.rs
impl Solution {
    pub fn maximum_difference(nums: Vec<i32>) -> i32 {
    let mut max: i32 = 0;
    let len: usize = nums.len();
    for i in 0..len-1 {
        for j in i+1..len {
            if max < nums[j] - nums[i] { max = nums[j] - nums[i];}
        }
    }
    return if max == 0 { -1 } else { max };
    }
}
